airbyte_cdk/sources/declarative/concurrent_declarative_source.py

```diff
@@ -171,8 +171,8 @@ def __init__(
         self,
         catalog: Optional[ConfiguredAirbyteCatalog],
         config: Optional[Mapping[str, Any]],
-        state: TState,
         source_config: ConnectionDefinition,
+        state: Optional[List[AirbyteStateMessage]] = None,
```

Comment:
aside - 
Noting here: arg order had to change because `source_config` (aka the manifest) _is_ required and we can't have required positional args after non-required ones.

Ideally, we'd use "*," to force these all as keyword-only args, but that seems overkill here for what we're doing. Note though that if any callers are passing a positional-only list of args here, those calls could break.